src/sentry/static/sentry/app/components/charts/utils.tsx

```diff
@@ -8,9 +8,11 @@ import {escape} from 'app/utils';
 const DEFAULT_TRUNCATE_LENGTH = 80;
 
 // In minutes
-const THIRTY_DAYS = 43200;
-const TWENTY_FOUR_HOURS = 1440;
-const ONE_HOUR = 60;
+export const THIRTY_DAYS = 43200;
+export const ONE_WEEK = 10080;
+export const TWO_WEEKS = 20160;
```

Comment:
Small nit but these should be flipped

```suggestion
export const TWO_WEEKS = 20160;
export const ONE_WEEK = 10080;
```